xbmc/addons/AddonDatabase.cpp

```diff
@@ -609,8 +609,9 @@ bool CAddonDatabase::GetRepositoryContent(const std::string& id, VECADDONS& addo
 
     {
       std::string sql = PrepareSQL(
-          " SELECT * FROM addons"
+          " SELECT addons.*, repo.addonID AS repoID FROM addons"
           " JOIN addonlinkrepo ON addons.id=addonlinkrepo.idAddon"
+          " JOIN repo ON repo.id=addonlinkrepo.idRepo"
           " WHERE addonlinkrepo.idRepo IN (%s)"
           " ORDER BY addons.addonID", StringUtils::Join(repoIds, ",").c_str());
```

Comment:
missed to change order, coming in with the next push
```suggestion
          " ORDER BY repo.addonID, addons.addonID", StringUtils::Join(repoIds, ",").c_str());
```